src/GraphQl/Type/FieldsBuilder.php

```diff
@@ -329,7 +337,7 @@ private function convertFilterArgsToTypes(array $args): array
         foreach ($args as $key => $value) {
             if (strpos($key, '.')) {
                 // Declare relations/nested fields in a GraphQL compatible syntax.
-                $args[str_replace('.', '_', $key)] = $value;
+                $args[str_replace('.', '__', $key)] = $value;
```

Comment:
Why not using the nesting operator twice here?

Reply:
That's a leftover, I'll update